pkg/strvals/parser_test.go

```diff
@@ -255,6 +255,10 @@ func TestParseSet(t *testing.T) {
 			str:    "nested[1][1]=1",
 			expect: map[string]interface{}{"nested": []interface{}{nil, []interface{}{nil, 1}}},
 		},
+		{
+			str:    "escapes=hello\\n\\tworld",
+			expect: map[string]interface{}{"escapes": "hello\n\tworld"},
+		},
```

Comment:
Helm used a `\\` to escape things today. That means, `\\\n` would become `\n`. The test would pass without any other changes if the test were re-written as:

```
{
	str:    "escapes=hello\\n\\tworld",
	expect: map[string]interface{}{"escapes": "hello\n\tworld"},
},
```

Is there a reason for the other changes?

Reply:
If we document the escape sequences I think we can close this.

Reply:
It is unclear what you have changed in your test. Do you mean (three `\`):
```
{
	str:    "escapes=hello\\\n\\\tworld",
	expect: map[string]interface{}{"escapes": "hello\n\tworld"},
},
```

Reply:
@mattfarina I don't know of any way to have a tab or newline currently (except passing the char directly).